inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -258,7 +277,9 @@ IExecutableNetworkInternal::Ptr clDNNEngine::LoadExeNetworkImpl(const InferenceE
     auto transformedNetwork = CloneAndTransformNetwork(network, conf);
     {
         OV_ITT_SCOPED_TASK(itt::domains::CLDNNPlugin, "clDNNEngine::LoadExeNetworkImpl::CreateExeNetwork");
-        return std::make_shared<CLDNNExecNetwork>(transformedNetwork, context, conf);
+        CLDNNExecNetwork::Ptr exeNetwork = std::make_shared<CLDNNExecNetwork>(transformedNetwork, context, conf);
+        UpdateStatistics(context);
```

Comment:
What about multiple calls of GetMetric for executable network? Does it still accumulate memory consumption for all networks or it returns per-network value?

Reply:
It still accumulates memory usages for networks allocated through the same context. So way to separate the memory usage of networks sharing the same context is currently under design and PoC, and the purpose of this PR is to query the overall memory usage statistics of all networks allocated through different contexts or the same context. BTW what does "multiple calls of GetMetric for executable network" mean? Could you please elaborate?

Reply:
I meant GetMetric() call for multiple networks which are loaded sequentially